### DISCUSSION TITLE
Cannot Sign in Git Desktop

### DISCUSSION THREAD

**Discussion by tuanson-afk** [community/community#146883] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hi, I just clicked the "continue with browser" to sign in with Git Destop but the arrow was rolling repeatedly so I can not sign in, please help me
![Lỗi](https://github.com/user-attachments/assets/82a5a464-9451-40fa-801d-3cd4a35e1ca0)
![Lỗi](https://github.com/user-attachments/assets/ebb5af0c-1119-4192-8e65-b31dd2957431)

**Comment by Lut-Lat-Aung** [comment]
It is trying to open browser (FireFox or Chrome or Brave) to continue sing in. Maybe check the browser in the background. But by the looks of it, you already sign in. I see your profile picture at the bottom left. Maybe just click cancel and restart the github desktop. If this still persist, you can refer to this link - https://docs.github.com/en/desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github-in-github-desktop.

I hope you find this answer helpful.

**Reply by tuanson-afk** [reply]
still not working :(((

**Reply by Lut-Lat-Aung** [reply]
Maybe uninstall Github Desktop and reinstall and sign in from the start. When you click "continue with browser", the browser doesn't show?

**Reply by tuanson-afk** [reply]
that's right, It's doesn't show any message or noti, just show like normal

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @tuanson-afk ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category, General, for you. Good luck!
